doc/data/messages/t/too-many-public-methods/good.py

```diff
@@ -1 +1,9 @@
-# This is a placeholder for correct code for this message.
+class Foo:
```

Comment:
Could you use an example of a messy API instead ? chatGPT can help to generate examples if you don't have inspiration.

Reply:
Fixed